pagopa/io-app-design-system
# [IOAPPX-408] Fix alignment of `Alert` icon when there's a line of content

## Short description
This PR fixes the alignment of `Alert` icon when there's just a line of content

## List of changes proposed in this pull request
- Add `onTextLayout` event to count the lines and apply different style based on context

### Preview
| Before | After |
|--------|--------|
| ![alert-oneline-before](https://github.com/user-attachments/assets/e7954c7f-2835-47df-9df4-58bf20bb6838) | ![alert-oneline-after](https://github.com/user-attachments/assets/d8ea5e1a-37e4-431c-b550-beab11c669aa) | 

## How to test
1. Launch the example app
2. Go to the **Alert** page and check if everything is OK
